src/cmd/cli/command/compose.go

```diff
@@ -90,7 +90,7 @@ func makeComposeUpCmd() *cobra.Command {
 				}
 
 				if !resp.Gpu {
-					return ErrNoPermission("usage of GPUs. To resolve see https://s.defang.io/subscription")
+					// return ErrNoPermission("usage of GPUs. To resolve see https://s.defang.io/subscription")
```

Comment:
Was this intentional? Let's remove the line above and below if we don't want to return an error here.

But if we don't want to return an error here, what's taking its place?

Reply:
Good catch. Reverted